.github/workflows/template_launchdarkly_code_references.yml

```diff
@@ -0,0 +1,47 @@
+name: Find LaunchDarkly flag code references
+
+on:
+  workflow_call:
+    inputs:
+      project-key:
+        required: false
+        type: string
+        default: 'default'
+    secrets:
+      access-token:
+        required: true
+
+jobs:
+  launchDarklyCodeReferences:
+
+    name: Find LaunchDarkly flag code references
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Create Files
+        run: |
+          mkdir .launchdarkly
+          cat <<EOT >> .launchdarkly/coderefs.yml
+          aliases:
+            - type: camelcase
+            - type: command
+              command: "./.launchdarkly/flag-alias.sh"
+              timeout: 5
+          EOT
+          cat <<EOT >> .launchdarkly/flag-alias.sh
+          #!/bin/sh
+          read flagKey <&0; echo "[\"\$flagKey\",\"\$(echo \$flagKey | awk '{print tolower(\$0)}' | awk -F'-' '{for(i=1;i<=NF;i++){\$i=toupper(substr(\$i,1,1)) substr(\$i,2)}} 1' OFS='' | awk '{print tolower(substr(\$0,0,1))substr(\$0,2)}')\"]"
+          EOT
+          chmod +x .launchdarkly/flag-alias.sh
+      - name: Show Files
+        run: |
+          cat .launchdarkly/coderefs.yml
+          cat .launchdarkly/flag-alias.sh
+      - name: LaunchDarkly Code References
+        uses: launchdarkly/find-code-references@v2.10.0
+        with:
+          accessToken: ${{ secrets.access-token }}
+          projKey: ${{ inputs.project-key }}
+          debug: true
```

Comment:
Is this debug option also necessary for the real runs? 🤔

Reply:
It is quite handy and it creates only 3 more log lines. From my point of view also the most important one for the workflow run ⤵️

```
DEBUG: 2023/06/08 13:25:55 client.go:589: [DEBUG] GET https://app.launchdarkly.com/api/v2/code-refs/repositories/pikasso
                  FLAG                 | # REFERENCES  
---------------------------------------+---------------
  test                                 |          987  
  pik-[19](https://github.com/Staffbase/pikasso/actions/runs/5211762155/jobs/9404468506#step:6:20)13-text-editor-module          |           15  
  ED-484-personalization-msft          |           11  
  feed-32[20](https://github.com/Staffbase/pikasso/actions/runs/5211762155/jobs/9404468506#step:6:21)-check-for-undefined-blocks |            7  
  ed-397-email-designer-tables         |            4  
  ed-400-text-editor-features          |            2  
  pik-2070-toolbar-sub-menu            |            2  
  ed-4[21](https://github.com/Staffbase/pikasso/actions/runs/5211762155/jobs/9404468506#step:6:22)-bugfix-draftjs-autosave       |            1  
  cwins-36-customization-headers       |            1  
  ed-4[27](https://github.com/Staffbase/pikasso/actions/runs/5211762155/jobs/9404468506#step:6:28)-quote-block-styling           |            1  
  ED-477-custom-font-caching           |            1  
  Other flags                          |            0
```